specification/resourceconnector/resource-manager/Microsoft.ResourceConnector/preview/2022-04-15-preview/appliances.json

```diff
@@ -0,0 +1,986 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-04-15-preview",
+    "title": "appliances",
```

Comment:
Is every file in this PR new changes? If any of them are from previous API version, please follow this: The first commit needs to be the previous api version and the new changes should only be added in the subsequent commits. This is explained in detail here (If using Open API which is recommended) [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version) or here(For manual process) : [https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/83/Manual-Process-Adding-new-API-version) . Please remove the "ARMChangesRequested" tag once you have fixed this issue.